### ISSUE TITLE
One API call for multiple rendering

### ISSUE THREAD

**Issue by SquallX** [tabalinas/jssocials#75]
Hi,

How can I get one call from count url if I display 2 instances (for the same url) in my page ?
Thanks you.

**Comment by tabalinas** [comment]
So you want the share control to request the count only once? 
Do you have real performance issue? I'm not sure having two instead of one call should make a huge difference, the other case if you have lots of shares...

There is no straightforward way to do it, but you could redefine `_loadCount` method something like:

```js
var counts = {};

$("#jssocials").jsSocials({
    _loadCount: function(share) {
         var d = $.Deferred();
         var countUrl = this._getCountUrl(share);
         
         if(count[countUrl]) {
             d.resolve(count[countUrl])
         } else {
             jsSocials.Socials.prototype._loadCount.apply(this, arguments).done(function(result) 
                 d.resolve(result);
             });
         }
         return d.promise();
    }
});
```

I have not tested the code above, just showing the main principle.
